### PR TITLE
mix two posts

### PR DESCRIPTION
**Pull request by InakiRaba91**

### CODE REVIEW

File: content/posts/continuous_training/building_training_pipeline.md [hunk 1]
```diff
@@ -106,7 +157,7 @@ The model is now ready to be used for inference. To test it on an image, simply
 poetry run python -m animal_classifier inference 1.png base_model.pth --frames-dir data/cats_and_dogs/frames --model-dir models/cats_and_dogs --threshold 0.5
```
Comment (mmeendez8) [inline content/posts/continuous_training/building_training_pipeline.md:157]: When you say "make sure you have poetry installed", i would add a link to poetry installation website because you describe what react is what nodejs but you assume poetry is known  (it is way less popular)

File: content/posts/continuous_training/building_training_pipeline.md [hunk 2]
```diff
@@ -130,6 +181,7 @@ So after training our model, making sure it outperforms the baseline and gaining
 </figure>
 
 Bear in mind that our model is simply an artifact, a file that contains the weights of the model. In order to make predictions, we need to wrap it in an API. We will use the <a href="https://pytorch.org/serve/">`torchserve`</a> library. The process goes as follows:
+
 1. **Model Archiver**: we need to create a `.mar` file that contains the model and the code to run it:
```
Comment (mmeendez8) [inline content/posts/continuous_training/building_training_pipeline.md:185]: Code formatting for the config.properties file looks weird

Comment (mmeendez8) [reply content/posts/continuous_training/building_training_pipeline.md:185]: Also the code formatting in the code section is not working as expected:
Note: we have reformated the `config.properties` for clarity. However, the model_snapshot must be a single line without spaces, as in the original [file](https://github.com/InakiRaba91/animal_classifier/blob/main/config.properties).

File: content/posts/continuous_training/building_training_pipeline.md [hunk 3]
```diff
@@ -198,6 +250,83 @@ We can now send requests to the endpoint and get predictions back:
     curl -X POST http://localhost:8080/predictions/animal -T ./data/cats_and_dogs/frames/1.png & curl -X POST http://localhost:8080/predictions/animal -T ./data/cats_and_dogs/frames/2.png
     ```
 
-# 4. References
+## 3.4. Proxy Deployment
+
+There's one last issue though. When we try to hit the torchserve endpoint from the labelling app, we get an error indicating it
+has been blocked by CORS policy. We will solve it by adding a FastAPI server that acts as a proxy to the torchserve endpoint. This will come handy since it gives as a WebUI to interact with the model. Furthermore, we will also leverage it for canary rollouts as we will see in a future post.
```
Comment (mmeendez8) [inline content/posts/continuous_training/building_training_pipeline.md:256]: Maybe a link that describes what CORS error is?

File: content/posts/continuous_training/building_training_pipeline.md [hunk 4]
```diff
@@ -198,6 +250,83 @@ We can now send requests to the endpoint and get predictions back:
...
+
+Then, run the following command to start the services:
+
+```bash
+docker-compose up -d
+```
+
+And go to `http://localhost:3000/` in your browser. You will be able get predictions back from the model and review them. 
+
+<figure class="figure" style="text-align: center;">
```
Comment (mmeendez8) [inline content/posts/continuous_training/building_training_pipeline.md:325]: Not displayed

File: content/posts/continuous_training/building_training_pipeline.md [hunk 5]
```diff
@@ -36,18 +36,64 @@ We have the following agents/components:
...
+
+# 2. Data collecetion
+
+A key element of any machine learning project is the **data**. At its core, AI models ingest vast amounts of data and are tasked with recognizing patterns and finding insights. Assuming the data used for training was representative of the real world, one can then expect the trained model to make accurate predictions on unseen scenarios.
+
+But how do you get that data? Well, you may be able to use publicly available datasets. However, you may find it hard to build a moat around your business if you are using the same data as everyone else. Moreover, it is unlikely that you will be able to find a dataset that is perfectly suited to your specific problem. In many cases, you will need to collect your own data.
+
+<figure class="figure" style="text-align: center;">
+  <img src="/building_labelling_app/labelling_app.svg" alt="Labelling App" width="100%" style="display: block; margin: auto;">
+  <figcaption class="caption" style="font-weight: normal; max-width: 80%; margin: auto;">Diagram for the Continuous Training System we will be exploring in this series. It highlights the labelling componente we will focus on.</figcaption>
```
Comment (mmeendez8) [inline content/posts/continuous_training/building_training_pipeline.md:53]: typo: componente -> component

File: content/posts/continuous_training/building_training_pipeline.md [hunk 6]
```diff
@@ -106,7 +157,7 @@ The model is now ready to be used for inference. To test it on an image, simply
 poetry run python -m animal_classifier inference 1.png base_model.pth --frames-dir data/cats_and_dogs/frames --model-dir models/cats_and_dogs --threshold 0.5
 ```
 
-## 2.2. Evaluation
+## 3.2. Evaluation
 
 Before deploying the model, we need to evaluate its performance. For that purpose, we have an evaluation module that receives a model and a dataset. It computes a summary metric, the average binary cross-entropy loss. We will carry out two steps:
```
Comment (mmeendez8) [inline content/posts/continuous_training/building_training_pipeline.md:162]: cross entropy loss as metric is weird?
